### PR TITLE
Ansible improvements to support more addons and better cert/token handling

### PR DESCRIPTION
**Pull request by eparis**

### CODE REVIEW

**Comment by eparis** [comment]
@jeffbean

File: contrib/ansible/roles/node/tasks/main.yml [hunk 1]
```diff
@@ -14,29 +14,53 @@
 - include: centos.yml
   when: not is_atomic and ansible_distribution == "CentOS"
 
+- name: Make sure manifest directory exists
+  file: path={{ kube_config_dir }}/manifests/ state=directory
```
Comment (jeffbean) [inline contrib/ansible/roles/node/tasks/main.yml]: should we make a manifests variable as well? to stay consistent

Comment (eparis) [inline contrib/ansible/roles/node/tasks/main.yml]: fixed.

File: contrib/ansible/roles/kubernetes/tasks/gen_tokens.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
```
Comment (jeffbean) [inline contrib/ansible/roles/kubernetes/tasks/gen_tokens.yml]: You forgot to copy the script to the host. My run fails becasue the /kube-gen-token.sh is not there.

Comment (eparis) [inline contrib/ansible/roles/kubernetes/tasks/gen_tokens.yml]: fixed

**Comment by jeffbean** [comment]
So everything looks good. 

The  only thing is the CentOS testing repo still only has Kubernetes 0.18 so the controller-manager flag `--root_ca_file` does not exist. 

I would say lets not get into the game of conditionalizing the flags based on the version of Kubernetes until we hit v1.0 and move forward with this.

**Comment by zmerlynn** [comment]
LGTM from a "not going to break core".

cc @quinton-hoole @thockin for rubber stamps. Low risk `contrib/` change.

**Comment by thockin** [comment]
LGTM  @quinton-hoole

**Comment by quinton-hoole** [comment]
LGTM.  ok-to-merge.

**Pull request merged**